docs/concepts-guidelines.md

```diff
@@ -144,7 +144,7 @@ Pull requests created by the action using the default `GITHUB_TOKEN` cannot trig
 
 There are a number of workarounds with different pros and cons.
 
-- Use the default `GITHUB_TOKEN` and allow the action to create pull requests that have no checks enabled. Manually close pull requests and immediately reopen them. This will enable `on: pull_request` workflows to run and be added as checks.
+- Use the default `GITHUB_TOKEN` and allow the action to create pull requests that have no checks enabled. Manually close pull requests and immediately reopen them. This will enable `on: pull_request` workflows to run and be added as checks. (Adding `draft: true` and a note about a close-and-reopen cycle to the message in `body` may help prevent human merges of pull requests where needed workflows have not yet been run.)
```

Comment:
Using draft PRs is a good tip. 🏅 

I'm conscious of adding too much detail to the documentation, so I would just like to simplify the tip a bit.
```suggestion
- Use the default `GITHUB_TOKEN` and allow the action to create pull requests that have no checks enabled. Manually close pull requests and immediately reopen them. This will enable `on: pull_request` workflows to run and be added as checks. Alternatively, create draft pull requests with `draft: true`. When the pull request is opened for review, checks will run.
```

Reply:
> I'm conscious of adding too much detail to the documentation, so I would just like to simplify the tip a bit.

Hi @peter-evans, I understand and it's not totally unexpected. I am happy to adjust but:
- I tried earlier today myself: Just pressing the "Ready to review" button does not seem to trigger any `on: pull_request` workflows (while that was, what I had originally hoped for). It needed a closed and re-open and the suggested simplification reads as if the pressing of "Ready to review" would be the key and sufficient. Am I missing something?
- I would want to replace "create draft" by "create [a] draft"

How would you like to continue?

Reply:
I just assumed pressing "Ready to review" is a state change for the PR from `draft` to `open` and would kick off checks, but clearly not if you've tried it.

Ok, let's just go with your original text then, since the tip should be focussed on preventing human error.